README.md

```diff
@@ -113,21 +107,22 @@ if __name__ == "__main__":
 
 *To execute:*
 
-    $ python hello_world_test.py -c sample_config.json
+    $ python hello_world_test.py -c sample_config.yml
 
 *Expect*
 
 A "Hello World!" toast notification appears on your device's screen.
 
-Within SampleTestBed, we used `"AndroidDevice" : "*"` to tell the test runner to
-automatically find all connected Android devices. You can also specify
-particular devices by serial number and attach extra attributes to the object:
+Within SampleTestBed's `Controllers` section, we used `"AndroidDevice" : "*"` to tell
+the test runner to automatically find all connected Android devices. You can also
+specify particular devices by serial number and attach extra attributes to the object:
 
-```python
-"AndroidDevice": [
-  {"serial": "xyz", "phone_number": "123456"},
-  {"serial": "abc", "label": "golden_device"},
-]
+```yaml
+AndroidDevice:
+  - serial: xyz,
```

Comment:
No need to change anything, but github's code review markers make yaml confusing to review..